code/addons/actions/src/stories/basics.stories.tsx

```diff
@@ -0,0 +1,19 @@
+// TODO -- for now react, going to generalise
+import React, { FC } from 'react';
+
+import { action } from '@storybook/addon-actions';
+
+// TODO -- this needs to be a generic component
+const Button: FC<{ onClick: () => void }> = ({ onClick, children }) => (
+  <button type="button" onClick={onClick}>
+    {children}
+  </button>
+);
+
+export default {
+  component: Button,
+};
+
+export const BasicExample = {
+  args: { onClick: action('hello-world') },
+};
```

Comment:
What if each renderer would export basic components, like `renderers/react/src/example/components` would contain `Button.ts`, or maybe even js/ts folders with their own versions of components?

Then each addon's stories just assumed there will be a component, like `addons/actions/src/examples/stories` would contain the code below, becoming scalable as it abstracts the implementation details of the component:

```suggestion
import { action } from '@storybook/addon-actions';
import { Button } from './Button'

export default {
  component: Button,
};

export const BasicExample = {
  args: { onClick: action('hello-world') },
};
```

Reply:
@yannbf we are doing something similar to this but the plan is to put the components on `global`.